cmd/neofs-adm/internal/modules/morph/initialize.go

```diff
@@ -363,14 +363,28 @@ loop:
 // If sysFee is -1, it is calculated automatically. If tryGroup is false,
 // global scope is used for the signer (useful when working with native contracts).
 func (c *initializeContext) sendCommitteeTx(script []byte, sysFee int64, tryGroup bool) error {
-	tx, err := c.Client.CreateTxFromScript(script, c.CommitteeAcc, sysFee, 0, []rpcclient.SignerAccount{{
+	cosigners := []rpcclient.SignerAccount{{
 		Signer:  c.getSigner(tryGroup),
 		Account: c.CommitteeAcc,
-	}})
+	}}
+	tx, err := c.Client.CreateTxFromScript(script, c.CommitteeAcc, sysFee, 0, cosigners)
 	if err != nil {
 		return fmt.Errorf("can't create tx: %w", err)
 	}
 
+	tx.Attributes = append(tx.Attributes, transaction.Attribute{Type: transaction.HighPriority})
+
+	// Calculate network fee again, because tx size has changed.
+	_, accounts, err := getSigners(c.CommitteeAcc, cosigners)
```

Comment:
If you're updating NeoGo to 0.99.2 you can try using `rpcclient/actor` that will simplify a lot of this code.